### PR TITLE
Fix nav styling when logged in

### PR DESCRIPTION
**Pull request by RoyEJohnson**

### CODE REVIEW

**Comment by RoyEJohnson** [comment]
[Bug: Global nav visual design specs not matching mockup, inconsistent across logged in/out states](https://trello.com/c/J7ZRcayx/835-bug-global-nav-visual-design-specs-not-matching-mockup-inconsistent-across-logged-in-out-states)

File: tutor/resources/styles/components/top-nav-bar.less [hunk 1]
```diff
@@ -37,9 +37,18 @@
...
+
   a {
     color: #777;
     text-decoration: none;
+
+    &:hover,
+    &[aria-expanded="true"] {
+      color: #333;
+    }
   }
```
Comment (nathanstitt) [inline tutor/resources/styles/components/top-nav-bar.less]: I think the dropdown rules could be moved to the `dropdown-menu` file that's included at the bottom, that way all menu style rules are consolidated.

File: tutor/specs/components/navbar/__snapshots__/index.spec.jsx.snap [hunk 2]
```diff
@@ -93,7 +93,10 @@ exports[`Main navbar renders and matches snapshot 1`] = `
         />
         <span
           title="Menu and settings"
-        />
+        >
+          Hi 
+          
+        </span>
          
         <span
```
Comment (nathanstitt) [inline tutor/specs/components/navbar/__snapshots__/index.spec.jsx.snap]: Glad you figured out how to update the snapshots - I just realized I'd never mentioned them to you 😁

File: tutor/src/components/navbar/course-name.jsx [hunk 3]
```diff
@@ -1,24 +0,0 @@
...
-    const { course } = this.props;
-    if (!course) { return null; }
-
-    return (
-      <TutorLink to="dashboard" params={{ courseId: course.id }} className="course-name">
-        {course.name}
-      </TutorLink>
-    );
-  }
-}
```
Comment (nathanstitt) [inline tutor/src/components/navbar/course-name.jsx]: Hm, I wasn't aware that UX wanted us to remove the course name altogether, I thought we were only supposed to hide it on narrow screens.  @Fredasaurus can you clarify?

File: tutor/src/components/navbar/user-actions-menu.jsx [hunk 4]
```diff
@@ -100,6 +100,8 @@ export default class UserActionsMenu extends React.PureComponent {
 
     const course = Courses.get(this.props.courseId);
     const isConceptCoach = get(course, 'is_concept_coach', false);
+    const firstName = User.name ? User.name.replace(/ .*/, '') : '';
+
```
Comment (nathanstitt) [inline tutor/src/components/navbar/user-actions-menu.jsx]: How about we add a method to User for getting the first name.  Probably something like:

```js
@computed get firstName() {
  return this.first_name || User.name.replace(/ .*/, '') || '';
}
```

Then we can just use `User.firstName` from the jsx below.

Comment (nathanstitt) [inline tutor/src/components/navbar/user-actions-menu.jsx]: fyi - user model file is in `tutor/src/models/user.js`

File: tutor/src/components/navbar/course-name.jsx (hunk 3 above)
Comment (nathanstitt) [inline tutor/src/components/navbar/course-name.jsx]: also, if we do remove the course name then we can no longer click it to go to the dashboard for a course.

**Pull request closed without merging**